flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/ChangelogReducingState.java

```diff
@@ -65,16 +80,31 @@ public V get() throws Exception {
     @Override
     public void add(V value) throws Exception {
         delegatedState.add(value);
+        changeLogger.stateUpdated(delegatedState.get(), getCurrentNamespace());
```

Comment:
For interface semantics, using ```delegatedState.getInternal()``` is better? However, it's no big deal.

Reply:
You are right (it was changed in #15200  and now in master).